tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -2319,8 +2319,8 @@ Status MklLayoutRewritePass::MergePadWithConv2D(std::unique_ptr<Graph>* g,
       // Conv2D has only 1 output (at slot 0) and merged node also has only 1
       // output (at slot 0).
       const int kPadWithConv2DOutputSlot = 0;
-      CHECK_NOTNULL((*g)->AddEdge(new_node, kPadWithConv2DOutputSlot, e->dst(),
-                                  e->dst_input()));
+      (*g)->AddEdge(new_node, kPadWithConv2DOutputSlot, e->dst(),
```

Comment:
Just curious, why did you remove the check?

Reply:
Actually, if we replace CHECK with DCHECK here, since DCHECK is a no op, that will not work, since this code has to be active during runtime. I also find that in the "if" part does not have CHECK so I just removed the CHECK from the "else". If you think, it is necessary, then I can  assign a variable to it and DCHECK to that variable.

Reply:
It's okay. I don't think it's necessary. Thank you for the explanation! :)